booking-app/src/main/java/org/motechproject/bookingapp/repository/ScreeningDataService.java

```diff
@@ -21,6 +22,19 @@
     long countFindByDate(Range<LocalDate> dateRange);
 
     @Lookup
-    List<Screening> findByDateAndClinicId(@LookupField(name = "date") LocalDate date,
-                                          @LookupField(name = "clinic.id") Long clinicId);
+    List<Screening> findByDateAndClinicId(@LookupField(name = "date") LocalDate date, @LookupField(name = "clinic.id") Long clinicId);
+
+    @Lookup
+    List<Screening> findByDateAndClinicLocation(@LookupField(name = "date") Range<LocalDate> date,
```

Comment:
rename the lookup, so date will be at the end: findByClinicLocationAndDate, so you can just add " And Date" to the name of the lookup in LookupHelper